main.go

```diff
@@ -15,7 +14,15 @@ func main() {
 		for _, component := range devfile.Data.GetAliasedComponents() {
```

Comment:
`GetAliasedComponents()`, this method could be removed, It was required for devfile V1, for V2 name is a required field

Reply:
Thank you, removing the function